courses/models.py

```diff
@@ -900,3 +900,27 @@ class Meta:
 
     def __str__(self):
         return f"Paid Course Run - {self.course_run.courseware_id} by {self.user.name}"
+
+    @classmethod
+    def fulfilled_paid_course_run_exists(cls, user: User, run: CourseRun):
```

Comment:
generally, a class method should deal with only class variables and the outside ones, We could have moved it to api.py without having to resolve circular dependency too but this is fine for now. keep it as is.